packages/core/src/common/utils/jsUtils.ts

```diff
@@ -70,3 +70,10 @@ export function countDecimalPlaces(num: number) {
     }
     return p;
 }
+
+const uniqueCountForPrefix = new Map<string, number>();
+export function uniqueId(prefix: string) {
+    const curCount = uniqueCountForPrefix.get(prefix) || 0;
+    uniqueCountForPrefix.set(prefix, curCount + 1);
+    return `///${prefix}-${curCount}///`;
```

Comment:
why the slashes? IMO they don't signify much, and could be misleading, suggesting some kind of regex